### PR TITLE
{bio}[foss/2018b] SqueezeMeta v1.0.0 w/ Python 2.7.15

### LINKED ISSUE #9393: SqueezeMeta doesn't work because vendored (patched) tools were replaced with proper dependencies
See discussion in https://github.com/jtamames/SqueezeMeta/issues/46#issuecomment-559225636

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~#9377~ (CompareM), fixes #9393

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 31 out of 31 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/a3c2dddf5ebd177ec9d868b18cd4200e for a full test report.

File: easybuild/easyconfigs/s/SqueezeMeta/SqueezeMeta-1.0.0-foss-2018b-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+dependencies = [
+    ('Perl', '5.28.0'),  # provides XML::Parser
+    ('XML-LibXML', '2.0132', '-Perl-%(perlver)s'),
+    ('Python', '2.7.15'),
+    ('matplotlib', '2.2.3', versionsuffix),
+    ('Pysam', '0.15.1', versionsuffix),
+    ('R', '3.5.1', versionsuffix),
+    ('BLAST+', '2.7.1'),
+    ('BWA', '0.7.17'),
+    #('barrnap', '0.9'),
```
Comment (verdurin) [inline easybuild/easyconfigs/s/SqueezeMeta/SqueezeMeta-1.0.0-foss-2018b-Python-2.7.15.eb:30]: @boegel could you appease the style checker here, please?

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/s/SqueezeMeta/SqueezeMeta-1.0.0-foss-2018b-Python-2.7.15.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/s/SqueezeMeta/SqueezeMeta-1.0.0-foss-2018b-Python-2.7.15.eb]: fixed in aebd17a

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2604.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/9c5e3758d460ce73743f01b78431496d for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 40 out of 40 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/ffdadc3d9e057dd5c07a2de7836c4964 for a full test report.

**Pull request merged**